CMakeLists.txt

```diff
@@ -395,11 +395,11 @@ OCV_OPTION(WITH_OPENCL_D3D11_NV "Include NVIDIA OpenCL D3D11 support" WITH_DIREC
 OCV_OPTION(WITH_LIBREALSENSE "Include Intel librealsense support" OFF
   VISIBLE_IF NOT WITH_INTELPERC
   VERIFY HAVE_LIBREALSENSE)
-OCV_OPTION(WITH_VA "Include VA support" ON
-  VISIBLE_IF UNIX AND NOT ANDROID
+OCV_OPTION(WITH_VA "Include VA support" (X86_64 OR X86)
```

Comment:
Why do we need to restrict this for x86 only?

Reply:
Although VA-API is not locked to this platform, I'm not aware of implementations on other platforms (see https://trac.ffmpeg.org/wiki/HWAccelIntro for example). Since we don't verify it we should disable it by default to avoid potential problems.